src/translated_content/zh_CN/docs/developing-applications/building-blocks/configuration/howto-manage-configuration.md

```
@@ -0,0 +1,684 @@
+        print(f"Got key={configuration.items[0].key} value={configuration.items[0].value} version={configuration.items[0].version}")
+```
+
+{{% /codetab %}}
+
+{{% codetab %}}
+
+```go
+package main
+
+import (
+	"context"
```

markdownlint-cli2 (0.17.2)

Hard tabs
Column: 1

(MD010, no-hard-tabs)